src/screens/InfoScreen/InfoScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+import { Container, TextContainer, StyledHeaderText, StyledText } from './styledComponents/InfoScreenStyledComponents';
+
+const InfoScreen: React.FC = ({}) => (
```

Comment:
React.FC doesn't seem to be a superior pattern at a glance; what are the pros/justification of making this change?

Reply:
It has something to do with my code editor/linter issue. I am using the FC, but there is an issue importing React as react from 'react', so I had to type out React so it knew what to use for React.FC. 

That was a confusing sentance

Reply:
This is something I looked into heavily a while ago and tldr, there's no actual difference that would matter in most cases. Having said that, there is this: https://github.com/facebook/create-react-app/pull/8177 Also this: https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript. So my personal conclusion was to just always go with the other approach just in case there ever was an issue using the react.fc approach.